diff --git a/input/js/sensors.js b/input/js/sensors.js
index 58d4441..e86007f 100644
--- a/input/js/sensors.js
+++ b/input/js/sensors.js
@@ -40,14 +40,16 @@ Expression:  (pv:hasFeature('WindSpeed') or pv:hasFeature('AirPressure'))Change Type: CHANGE_IN_VARIATION_POINT_BODY
 		var measure = measureText.value;
 		var intValue = checkMeasure(min, max, measure);
 		if (isNaN(intValue)) return false;
+		var root = stratify(data)//Authored by: letimome in commit:e818ef7e0421a6c0659ffaf5b898907afa34b6ee, with message:product berlin changes
+	      .sum(function(d) { return d.value; })//Authored by: letimome in commit:e818ef7e0421a6c0659ffaf5b898907afa34b6ee, with message:product berlin changes
+	      .sort(function(a, b) { return b.height - a.height || b.value - a.value; });//Authored by: letimome in commit:e818ef7e0421a6c0659ffaf5b898907afa34b6ee, with message:product berlin changes
+
 		function getMetaContentByName(name,content){
 			   var content = (content==null)?'content':content;
 			   return document.querySelector("meta[name='"+name+"']").getAttribute(content);
 			}
 			
 			$(document).ready(function(){
-				//console.log(diffString); //diffString should be the value of diffview value
-				//get select("#diffvalue"). metadata
 				var diffvalue = getMetaContentByName("diffvalue");
 				if (diffvalue==null) return null;
 			});
