diff --git a/input/js/settings.js b/input/js/settings.js
index 0c4a643..8eace39 100644
--- a/input/js/settings.js
+++ b/input/js/settings.js
@@ -17,5 +17,5 @@ Expression: (pv:hasFeature('WindSpeed'))Change Type: CHANGE_IN_VARIATION_POINT_BODY
 //PV:IFCOND(pv:hasFeature('WindSpeed'))
 var minWind = 0;
-var maxWind = 130;
+var maxWind = 160;//Authored by: letimome in commit:27baca19643dfa22917427ec2f031992e387edfc, with message:multiple issues: #13 #14 #15 
 var windScale = 'km/h';
 //PV:ENDCOND
