diff --git a/input/js/sensors.js b/input/js/sensors.js
index 8a7711e..41367f5 100644
--- a/input/js/sensors.js
+++ b/input/js/sensors.js
@@ -14,8 +17,11 @@ Expression:  (pv:hasFeature('WindSpeed'))Change Type: CHANGE_IN_VARIATION_POINT_BODY
  var measureText = document.getElementById("w_measure");
  windMeasure = measureText.value;
  var pointer = document.getElementById("w_point");
- 
- applyTachoValue(minWind, maxWind, measureText, pointer);
+ if (measureText && pointer) {//Authored by: letimome in commit:daf2fe5d8a9b7884dccd4bf9c871a34d9e049f87, with message:resolved #11 #12 and a fix on main title 
+  var measure = measureText.value;//Authored by: letimome in commit:daf2fe5d8a9b7884dccd4bf9c871a34d9e049f87, with message:resolved #11 #12 and a fix on main title 
+  var intValue = checkMeasure(min, max, measure);//Authored by: letimome in commit:daf2fe5d8a9b7884dccd4bf9c871a34d9e049f87, with message:resolved #11 #12 and a fix on main title 
+ }else//Authored by: letimome in commit:daf2fe5d8a9b7884dccd4bf9c871a34d9e049f87, with message:resolved #11 #12 and a fix on main title 
+   applyTachoValue(minWind, maxWind, measureText, pointer);//Authored by: letimome in commit:daf2fe5d8a9b7884dccd4bf9c871a34d9e049f87, with message:resolved #11 #12 and a fix on main title 
  setWarnings();
  return false;
 }
